src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -180,4 +181,8 @@ public void openNotifications() {
     public void toggleLocationServices() {
         CommandExecutionHelper.execute(this, toggleLocationServicesCommand());
     }
+
+    public String getCurrentPackage() {
```

Comment:
I think it would be better to move this method to the `io.appium.java_client.android.StartsActivity` interface and to make it `default`.